_data/meltano/utilities/tableau/tableau.yml

```
@@ -0,0 +1,52 @@
+commands:
+  refresh:
+    args: refresh $TABLEAU_DATASOURCE_LUID
+    description: Refresh a Tableau datasource.
+definition: |
+  Tableau is a widely used data visualisation and exploration tool.
+
+description: A modern, enterprise-ready business intelligence tool.
+docs: https://github.com/gabilew/tableau-ext
+domain_url: https://www.tableau.com/
+executable: tableau_invoker
+ext_repo: https://github.com/gabilew/tableau-ext
+keywords:
+- analysis
+label: Tableau
+logo_url: /assets/logos/utilities/tableau.png
+maintenance_status: active
+name: tableau
+namespace: tableau
+next_steps: |
+  1. For username/password auth you can skip this step but for API token auth you will need
+  to generate an API token on your Tableau server to authenticate any requests:
+  [Tableau API Auth Docs](https://help.tableau.com/current/api/rest_api/en-us/REST/rest_api_concepts_auth.htm)
+pip_url: requests==^2.28.2
```

```suggestion
pip_url: git+https://github.com/gabilew/tableau-ext.git
```

I think the extension should be the only pip package required. If you install the ext repo you would get requests with it, right?

---

correct, will update thanks!